x/sequencer/keeper/fraud.go

```diff
@@ -64,17 +64,37 @@ func (k Keeper) SlashLiveness(ctx sdk.Context, rollappID string) error {
 
 	// correct formula is e.g. min(sequencer tokens, max(1, sequencer tokens * 0.01 ))
 
+	err := k.livenessSlash(ctx, &seq)
+	if err != nil {
+		return errorsmod.Wrap(err, "slash")
+	}
+	k.livenessDishonor(ctx, &seq)
+	k.SetSequencer(ctx, seq)
+	return nil
+}
+
+func (k Keeper) livenessSlash(ctx sdk.Context, seq *types.Sequencer) error {
 	mul := k.GetParams(ctx).LivenessSlashMinMultiplier
 	abs := k.GetParams(ctx).LivenessSlashMinAbsolute
 	tokens := seq.TokensCoin()
 	tokensMul := ucoin.MulDec(mul, tokens)
 	amt := ucoin.SimpleMin(tokens, ucoin.SimpleMax(abs, tokensMul[0]))
-	err := errorsmod.Wrap(k.slash(ctx, &seq, amt, sdk.ZeroDec(), nil), "slash")
-	k.SetSequencer(ctx, seq)
-	return err
+	return errorsmod.Wrap(k.slash(ctx, seq, amt, sdk.ZeroDec(), nil), "slash")
+}
+
+func (k Keeper) livenessHonor(ctx sdk.Context, seq *types.Sequencer) {
+	reward := k.GetParams(ctx).DishonorStateUpdate
```

Comment:
`DishonorStateUpdate` should be called `HonorStateUpdate`?

Reply:
+1. confusing imo.

Reply:
maybe also better call it HonorLiveness ? to match the rest?

Reply:
is this a must, or a nit?
I prefer to keep fields with same noun prefix